go.mod

```diff
@@ -349,7 +350,10 @@ replace github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.1+in
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
 // needed to address mismatch between cosmosSDK and hdevalence/ed25519consensus
-replace filippo.io/edwards25519 => filippo.io/edwards25519 v1.0.0-beta.3
+replace filippo.io/edwards25519 => filippo.io/edwards25519 v1.0.0-rc.1
+
+// updating CosmWasm to v1.0.0 which brings ARM support
+replace github.com/CosmWasm/wasmvm => github.com/CosmWasm/wasmvm v1.0.0
```

Comment:
Does `integration-tests/go.mod` need the same/similar?

Reply:
I don't think so. 

This is an indirect dependency of `github.com/smartcontractkit/chainlink-terra`, which isn't present in local `integration-tests`.

Reply:
Yeah, it looks it does. We need to untangle that dependency tree.